eq-author-api/app.js

```diff
@@ -35,6 +35,22 @@ db(process.env.DB_SECRET_ID)
 
     const authMiddleware = createAuthMiddleware(logger, context);
 
+    const versionMiddleware = (req, res, next) => {
+      if (
+        process.env.NODE_ENV !== "development" &&
+        req.headers.clientversion !== process.env.EQ_AUTHOR_API_VERSION
+      ) {
+        logger.info(
+          {
+            clientversion: req.headers.clientversion,
+            apiversion: process.env.EQ_AUTHOR_API_VERSION,
```

Comment:
Should we send this in all responses? Or is the status page enough. I think it probably is.